gpx-docs/docs/commands/Remove.md

```
@@ -1,19 +1,63 @@
+## Examples
+
+### Remove a non-active profile
+
+```bash
+gpx remove personal
 ```
-- Removes the profile: ```<non_active_profile>```
 
-### Remove active profile (blocked)
+Expected output:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)